cpp/include/raft/linalg/detail/strided_reduction.cuh

```diff
@@ -33,33 +33,54 @@ namespace detail {
 // for column major layout
 template <typename Type, typename MainLambda>
```

Comment:
Could you add a short note that describe that a compensated summation is performed? Do I see correctly that even block reduction is compensated, and only the final grid reduction is the place where we are not having additional compensation?